app/models/like.rb

```diff
@@ -1,2 +1,11 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_likes_count
+
+  private
+
+  def update_likes_count
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Kindly make sure the associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).👍
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```